search/src/Components/wrapper/reactive_search.js

```diff
@@ -163,7 +163,7 @@ class ReactiveSearchComponent extends Component {
 							<ReactiveList.ResultCardsWrapper>
 								{
 									data.map(item => (
-										<ResultCard key={item._id}>
+										<ResultCard key={item._id} href={esURL + item.dataset_url}>
```

Comment:
I note that esURL is now a bad name. Maybe now is a good time to change it to rootURL?

Reply:
Are you assured that there is a `/` at the end of esURL?

Is it sufficient to just have the following which should mean the reference is relative to the domain root?
```suggestion
										<ResultCard key={item._id} href={"/" + item.dataset_url}>
```

Reply:
this var name is everywhere for now, including other branches. Let's clean this together in another pr after merging together, maybe export it in App.js. using "/" perhaps will work but it needs to be tested on vm.